pom.xml

```diff
@@ -103,6 +103,16 @@
       <artifactId>commons-httpclient</artifactId>
       <version>3.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>19.0</version>
+    </dependency>
+    <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-collections4</artifactId>
+        <version>4.1</version>
+    </dependency>
```

Comment:
Hm, pulling in a new dependency to creating one list of strings is a bit overkill.
Could you keep the old version and maybe change it to Lists.asList() from guava